packaging/nuget/nservicebus.host/EndpointConfig.cs

```diff
@@ -7,9 +7,17 @@ public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(EndpointConfiguration endpointConfiguration)
         {
-            // NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. 
+            //TODO: NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence.
             // Refer to the documentation for more details on specific options.
             endpointConfiguration.UsePersistence<PLEASE_SELECT_ONE>();
+
+            //TODO: NServiceBus will move messages that fail repeatedly to a separate "error" queue. We recommend 
+            // that you start with a shared error queue for all your endpoints for easy integration with ServiceControl.
+            endpointConfiguration.SendFailedMessagesTo("error");
+
+            //TODO: NServiceBus will store a copy of each successfully process message in a separate "audit" queue. We recommend
```

Comment:
We have the TODO:s in the bootstrapper (I copy and pasted them from there :)  )

https://github.com/Particular/NServiceBus.Bootstrap.WindowsService/blob/develop/src/NServiceBus.Bootstrap.WindowsService/ProgramService.cs#L46

I'm happy with removing them. @SimonCropp since you added them do you have an opinion?